installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -92,6 +92,21 @@
                 PerMachine="yes"
                 UninstallCommand="/silent /uninstall">
             </ExePackage>
+            <ExePackage
```

Comment:
Is 3.1 runtime still needed?

Reply:
Probably not, I have it in mind, but leaving it for a follow-up PR